nucypher/utilities/prometheus/collector.py

```diff
@@ -121,6 +124,13 @@ def _collect_internal(self) -> None:
 
         self.metrics["learning_status"].state('running' if self.ursula._learning_task.running else 'stopped')
         self.metrics["known_nodes_gauge"].set(len(self.ursula.known_nodes))
+        if self.ursula._availability_tracker:
+            if self.ursula._availability_tracker.running:
```

Comment:
Used the same 
```
    if ursula._availability_tracker:
        if ursula._availability_tracker.running:
```
check from https://github.com/nucypher/nucypher/blob/76eea579a3b94e1ea2a8019601e0b454f81e5e2c/nucypher/cli/painting/nodes.py#L84-L85